privacy-on-beam/pbeam/aggregations_test.go

```diff
@@ -302,34 +302,30 @@ func TestBoundedSumFloat64FnExtractOutputWithSpecifiedPartitionsDoesNotThreshold
 
 		got := fn.ExtractOutput(accum)
 		if got == nil {
-			t.Errorf("ExtractOutput: for %s got: %f, want actual value", tc.desc, *got)
+			t.Errorf("ExtractOutput for %s thresholded with specified partitions when it shouldn't", tc.desc)
 		}
 	}
 }
 
-// Check elements with unspecified partitions are dropped.
+// TestDropUnspecifiedPartitionsVFn check that elements with unspecified partitions are dropped.
 // This function is used for count and distinct_id.
-func TestDropUnspecifiedPartitionsForCount(t *testing.T) {
+func TestDropUnspecifiedPartitionsVFn(t *testing.T) {
 	pairs := concatenatePairs(
-		makePairsWithFixedVStartingFromKey(0, 7, 0),
-		makePairsWithFixedVStartingFromKey(7, 52, 1),
-		makePairsWithFixedVStartingFromKey(7, 52, 1),
-		makePairsWithFixedVStartingFromKey(7+52, 99, 2),
-		makePairsWithFixedVStartingFromKey(7+52, 99, 2),
-		makePairsWithFixedVStartingFromKey(7+52, 99, 2),
-		makePairsWithFixedVStartingFromKey(7+52+99, 10, 3),
-		makePairsWithFixedVStartingFromKey(7+52+99, 10, 3),
-		makePairsWithFixedVStartingFromKey(7+52+99, 10, 3),
-		makePairsWithFixedVStartingFromKey(7+52+99, 10, 3),
+		makePairsWithFixedV(7, 0),
+		makePairsWithFixedV(52, 1),
+		makePairsWithFixedV(99, 2),
+		makePairsWithFixedV(99, 2),
+		makePairsWithFixedV(99, 2),
```

Comment:
Do you need to add these 3 times? What are you testing by this? If this is not required, please simply do it once.